src/analysis/retail/warlock/affliction/modules/features/AlwaysBeCasting.tsx

```diff
@@ -38,39 +39,9 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
             message: `${formatPercentage(actual)}% downtime`,
           }),
         )
-        .recommended(`<${formatPercentage(recommended)}% is recommended`),
-    );
-  }
-
-  statistic() {
-    return (
-      <Statistic
-        position={STATISTIC_ORDER.CORE(2)}
-        tooltip={
-          <>
-            Downtime is available time not used to cast anything (including not having your GCD
-            rolling). This can be caused by delays between casting spells, latency, cast
-            interrupting or just simply not casting anything (e.g. due to movement/stunned).
-            <br />
-            <ul>
-              <li>
-                You spent <strong>{formatPercentage(this.activeTimePercentage)}%</strong> of your
-                time casting something.
-              </li>
-              <li>
-                You spent <strong>{formatPercentage(this.downtimePercentage)}%</strong> of your time
-                casting nothing at all.
-              </li>
-            </ul>
-          </>
-        }
-      >
-        <div className="pad">
-          <label>Active time</label>
-
-          <Gauge value={this.activeTimePercentage} />
-        </div>
-      </Statistic>
```

Comment:
what is the motivation behind removing this? was it causing errors? or is this now a part of the superclass and not needed here?

Reply:
I picked a spec at random (fury) and saw it didn't exist. I see that it exists in a couple of others so I'll get this added back. Thanks!